t04_refactored.py

```diff
@@ -218,9 +220,46 @@ def team_14_adv(username, delay):
 
 
 def team_15_adv(username, delay):
-    pass
+
+    """
+    # https://docs.google.com/document/d/1Xi3PVM3_64dIIpbPdPyi13yFwu7lCRqUi7PjU0Fy0hI/edit?usp=sharing
+    #Tobore Takpor
+    #Jack Ngog
+    """
+
+
     # TODO Add your code here
     return True
+
+from time import sleep
+opponent = input("Who will you choose for your opponent? (Scott/Tojo/Bystander)")
+delay = 1
```

Comment:
`delay` is an input parameter, so it does not need defined again in the function. This overwrites the input parameter, rendering it obsolete.